lua/wikis/commons/Condition.lua

```diff
@@ -174,10 +174,52 @@ function ColumnName:toString()
 	return self.name
 end
 
+local ConditionUtil = {}
+
+---Builds "matches any of" condition from the given collection of values.
+---@param column ColumnName
+---@param values (string|number)[]
+---@return ConditionTree?
+function ConditionUtil.anyOf(column, values)
+	return ConditionUtil._multiValueCondition(column, BooleanOperator.any, Comparator.equals, values)
+end
+
+---Builds "matches all of" condition from the given collection of values.
+---@param column ColumnName
+---@param values (string|number)[]
+---@return ConditionTree?
+function ConditionUtil.allOf(column, values)
+	return ConditionUtil._multiValueCondition(column, BooleanOperator.all, Comparator.equals, values)
+end
```

Comment:
What's the usecase of this one?
It seems like this condition won't make any sense (if one value matches, others cannot, unless they "modify" the equals with ! or <>)

Reply:
yeah the comparator needs to be inputtabel for this one